operate/bridge/providers/native_bridge_provider.py

```diff
@@ -75,13 +79,15 @@ def __init__(
     ) -> None:
         """Initialize the bridge contract adaptor."""
         super().__init__()
+        if from_chain == to_chain:
+            raise ValueError("from_chain and to_chain cannot be the same.")
```

Comment:
does the bridge itself doesn't support that? Can we use this module to simply swap?